cocos/scripting/lua-bindings/manual/CCLuaEngine.cpp

```diff
@@ -439,13 +439,13 @@ int LuaEngine::handleCommonEvent(void* data)
         return 0;
    
     CommonScriptData* commonInfo = static_cast<CommonScriptData*>(data);
-    if (NULL == commonInfo->eventName || 0 == commonInfo->handler)
+    if ('\0' == commonInfo->eventName[0] || 0 == commonInfo->handler)
```

Comment:
@xpol , why to change this line ?
Before:
If eventName is empty string, this line will not return directly.
Now:
If eventName is empty string, this line will return directly.

It broke compatibility.